cmd/loadtest/loadtest.go

```diff
@@ -1187,6 +1187,12 @@ func loadTestRPC(ctx context.Context, c *ethclient.Client, nonce uint64, ia *Ind
 		_, err = c.SuggestGasPrice(ctx)
 	} else if funcNum < 21 {
 		log.Trace().Msg("eth_estimateGas")
+		var tx *ethtypes.Transaction
+		tx, _, err = c.TransactionByHash(ctx, ethcommon.HexToHash(ia.TransactionIDs[randSrc.Intn(len(ia.TransactionIDs))]))
+		if err != nil {
+			log.Error().Err(err).Msg("Unable to get the transaction hash")
```

Comment:
Should you `return` in this case rather than running `c.EstimateGas`?

Reply:
yup, I forgot to add `return` 🤦‍♂️